lib/webResource.ts

```diff
@@ -271,10 +286,11 @@ export class WebResource {
 export interface RequestPrepareOptions {
   method: HttpMethods;
   url?: string;
-  queryParameters?: { [key: string]: any } | ParameterValue;
+  queryParameters?: { [key: string]: string | ParameterValue };
```

Comment:
This is a pretty significant change (Changing from a type-less dictionary or a Parameter value to a dictionary with either string or ParameterValue values). Is it what you meant to do?

Reply:
See discussion https://github.com/Azure/ms-rest-js/pull/21#discussion_r178665476